### ISSUE TITLE
load data

### ISSUE THREAD

**Issue by SaidaSaad** [kenchan0226/keyphrase-generation-rl#16]
Hello 

I would like to ask about the possibility to train using only the absent Kps.
So can you please let me know how can I change in the function  load_data_and_vocab(opt, load_train=True) in case of "one to many"  to load only  absent KPs ?
Thanks

**Comment by kenchan0226** [comment]
You need to remove all the present keyphrases from the training data before running the preprocess.py. You need to remove the <peos> delimiter as well since you don't have present keyphrases.

**Comment by SaidaSaad** [comment]
Hello Kechan , Thanks you for your replies , I really appreciate.
I have question about the reinforcement that may be not relate to .  In case if I have another model that used to distinguish between the fake and true keypharse . This model I should use to evaluate the generate  keypharses regarding this . I face problem in design the reward function . because this model gave me hidden states, logits and loss for each batch. I do not know which one I should use to calculate the rewards for each batch. I hope you can help me in that.
